lib/TAP/Formatter/Color.pm

```diff
@@ -12,7 +12,7 @@ my $NO_COLOR;
 BEGIN {
     $NO_COLOR = 0;
 
-    eval 'use Term::ANSIColor';
+    eval 'use Term::ANSIColor qw//';
```

Comment:
either `eval { require Term::ANSIColor }` or `eval 'require Term::ANSIColor'` are fine; the latter should be used if `[AutoPrereqs]` (via Dist::Zilla) is employed, as the former construct will be parsed as a hard prerequisite unless it is modified with `[Prereqs::Soften]` (which I do in a few distributions, to turn the eval'd module into a 'suggests').